Consigment/Consigment/Consigment.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsigmentStruct
+{
+    public struct Consigment
+    {
+        int quantity;
+        public int Quantity
+        {
+            get => quantity;
+            set
+            {
+                if (value < 0)
+                    throw new ArgumentException("Количество единиц товара должно быть неотрицательным");
+                quantity = value;
+            }
+        }
+        double price;
+        public double Price
+        {
+            get => price;
+            set
+            {
+                if (value < 0)
+                    throw new ArgumentException("Количество единиц товара должно быть неотрицательным");
+                price = Math.Round(value, 2);
+            }
+        }
+        public double Cost
+        {
+            get => Math.Round((Price * Quantity),2);
+        }
```

Comment:
Можно еще уменьшить инфраструктурный код
```suggestion
        public double Cost => Math.Round((Price * Quantity),2);

```